scripts/pack/pack.js

```diff
@@ -0,0 +1,26 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+const fs = require('fs');
+const path = require('path');
+const shell = require('shelljs');
+
+const packageName = path.basename(process.cwd());
+const archiveName = `${packageName}.tgz`;
+
+// Remove the previous archive file if one exists.
+if (fs.existsSync(archiveName)) {
+  shell.rm(archiveName);
+}
+
+// Pack the package.
+shell.exec('npm pack --ignore-scripts', { silent: true }, (code, stdout, stderr) => {
+  if (code !== 0) {
+    /* eslint-disable-next-line no-console */
+    console.log(stderr);
+    shell.exit(-1);
+  }
+
+  // By default npm pack will append the package version to the tar archive file name.
+  // Rename the tar achive file to exclude the appended version.
+  shell.mv(stdout, archiveName);
```

Comment:
@StephenEsser What happens if two banches have different tarfiles checked in? `master` has the latest version and a 'test-enhancement' branch has a different version. How will this resolve?

Reply:
I think I take back my question, you would need to reference the branch here:
`"package-name": "https://raw.githubusercontent.com/org/project/blob/package-name.tgz"`